demisto_sdk/dev_tools/initiator.py

```diff
@@ -0,0 +1,261 @@
+        elif os.path.isfile('content-descriptor.json'):
+            self.full_output_path = os.path.join("Packs", self.name)
+
+        # if non of the above conditions apply - create the pack in current directory
+        else:
+            self.full_output_path = self.name
+
+        if not self.create_new_directory():
+            return False
+        for directory in self.DIR_LIST:
+            path = os.path.join(self.full_output_path, directory)
+            os.mkdir(path=path)
+
+        with open(os.path.join(self.full_output_path, 'CHANGELOG.md'), 'a') as fp:
+            fp.write("## [Unreleased]")
+
+        fp = open(os.path.join(self.full_output_path, 'README.md'), 'a')
+        fp.close()
+
+        with open(os.path.join(self.full_output_path, 'metadata.json'), 'a') as fp:
+            # TODO fill this once metadata.json script is ready
+            fp.write('[]')
+
+        fp = open(os.path.join(self.full_output_path, '.secrets-ignore'), 'a')
+        fp.close()
+
+        fp = open(os.path.join(self.full_output_path, '.pack-ignore'), 'a')
+        fp.close()
+
+        print_color(f"Successfully created the pack {self.name} in: {self.full_output_path}", LOG_COLORS.GREEN)
+
+        create_integration = str(input("\nDo you want to create an integration in the pack? Y/N ")).lower()
+        if create_integration == 'y':
+            self.name = ''
+            self.get_created_object_name(created_object="integration")
+            self.output_dir = os.path.join(self.full_output_path, INTEGRATIONS_DIR)
+            self.is_integration = True
+            return self.integration_init()
+
+        return True
+
+    def integration_init(self) -> bool:
+        """Creates a new integration according to a template.
+
+        Returns:
+            bool. True if the integration was created successfully, False otherwise.
+        """
+        # if output directory given create the integration there
+        if len(self.output_dir) > 0:
+            self.full_output_path = os.path.join(self.output_dir, self.name)
+
+        # will create the integration under the Integrations directory of the pack
+        elif os.path.isdir(INTEGRATIONS_DIR):
+            self.full_output_path = os.path.join('Integrations', self.name)
+
+        # if non of the conditions above apply - create the integration in the local directory
+        else:
+            self.full_output_path = self.name
+
+        if not self.create_new_directory():
+            return False
+
+        hello_world_path = os.path.normpath(os.path.join(__file__, "..", "..", 'common', 'templates',
+                                                         self.HELLO_WORLD_INTEGRATION))
+
+        copy_tree(str(hello_world_path), self.full_output_path)
+        if self.name != self.HELLO_WORLD_INTEGRATION:
+            self.rename(current_suffix=self.HELLO_WORLD_INTEGRATION)
+            self.yml_reformatting(current_suffix=self.HELLO_WORLD_INTEGRATION)
```

Comment:
shouldn't you use the name here?

Reply:
nope as I said this current_suffix is the current name - it will be overwritten by self.name in the function

Reply:
but you are doing this after the rename

Reply:
rename does not work on the yml file - exactly for that reason
also stated in the rename function's description: Renames the python, description, test and image file in the path to fit the newly created integration/script

Reply:
I'll add a note explaining this